### PR TITLE
Add Roach temperature code and table. Closes #180

### LINKED ISSUE #180: Start monitoring ROACH temperatures
As per HERA-Team/HERA_Commissioning#54. In-container temperatures are oscillating in a way that affects the data, and @jack-h has added monitoring points in the correlator redis database that record the temps in a better way than we have right now. We should start recording these data into M&C ASAP — this could be pretty important for H1C analysis going forward.

### PR DESCRIPTION
**Pull request by bhazelton**
This has been tested at site and it seems to work fine.

### CODE REVIEW

File: hera_mc/mc_session.py [hunk 1]
```diff
@@ -825,6 +825,72 @@ def get_weather_data(self, starttime, stoptime=None, variable=None):
                                  stoptime=stoptime, filter_column='variable',
                                  filter_value=variable)
 
+    def add_roach_temperature(self, time, roach, ambient_temp, inlet_temp,
+                              outlet_temp, fpga_temp, ppc_temp):
+        """
+        Add new roach temperature to the M&C database.
```
Comment (pkgw) [inline hera_mc/mc_session.py]: For posterity, it might help to define what a "roach" is.

File: hera_mc/mc_session.py [hunk 2]
```diff
@@ -825,6 +825,72 @@ def get_weather_data(self, starttime, stoptime=None, variable=None):
...
+
+        Parameters:
+        ------------
+        time: astropy time object
+            astropy time object based on a timestamp from the katportal sensor.
+        roach: int
+            roach number
```
Comment (pkgw) [inline hera_mc/mc_session.py]: Perhaps these should be strings? One could imagine a future where there's not a clear and easy numbering of the ROACHes.

File: hera_mc/roach.py [hunk 3]
```diff
@@ -0,0 +1,127 @@
...
+    redis_dict: A dict spoofing the return dict from _get_redis_dict for testing
+        purposes. Default: None
+
+    Returns:
+    -----------
+    A list of RoachTemperature objects
+    """
+
+    roach_obj_list = []
+    for i in range(1, 9):
```
Comment (pkgw) [inline hera_mc/roach.py]: This feels like something that shouldn't be hardcoded here.

File: hera_mc/mc_session.py (hunk 2 above)
Comment (bhazelton) [reply hera_mc/mc_session.py]: sure, I'll change it.

File: hera_mc/mc_session.py (hunk 1 above)
Comment (bhazelton) [reply hera_mc/mc_session.py]: I'll add a little more info.

File: hera_mc/roach.py (hunk 3 above)
Comment (bhazelton) [reply hera_mc/roach.py]: yeah, I agree, I just used some code from Jack. @jack-h can you suggest something better?

Comment (bhazelton) [reply hera_mc/roach.py]: It's not much better, but I moved the hardcoded list of roach names to the top of roach.py. I'm happy for other suggestions or we can just go with this.

**Pull request merged**